.github/ISSUE_TEMPLATE/monorepo-write.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Access Voucher
+about: Request write access to the Solana monorepo
+title: Request Access (Level 2) for [Username]
+labels: access
+assignees: joeaba
+
+---
+
+## Requesting Access Level 2 for [Username] to the Solana monorepo
```

Comment:
For level 2 requests, how about another `input` element that requires the user provide the link to approved level 1 issue as a part of their submission